HW0/P5/P5.txt

```diff
@@ -1 +1,6 @@
 # Your answers here.
+1. 255 
+2. yes; There will be 8 employees each counting 32 bags at the same time. 34s(31+log2(8))
+3. 8s(2^8=256)
+5. 15s
+6. 260;133;yes, for example, firstly give 16 cashiers each 16 bags.  
```

Comment:
I feel like an explanation or mathematical expression for this. I came up with a slightly different answer.